src/test/java/plugins/AntPluginTest.java

```diff
@@ -23,15 +32,74 @@ public class AntPluginTest extends AbstractJUnitTest {
     private static final String INSTALL_VERSION_1_10 = "1.10.5";
     private static final String INSTALL_NAME_1_10 = "ant_" + INSTALL_VERSION_1_10;
 
+    private static final String NATIVE_ANT_NAME = "native_ant";
+
+    private static final String OK_PROP1 = "okPROP1=foo_bar_ok_1";
+    private static final String OK_PROP2 = "okPROP2=foo_bar_ok_2";
+    private static final String NOK_PROP1 = "nokPROP1=foo_bar_nok_1";
+    private static final String NOK_PROP2 = "nokPROP2=foo_bar_nok_2";
+    private static final String PROPERTIES = OK_PROP1+"\n"+OK_PROP2+"\n"+NOK_PROP1+"\n"+NOK_PROP2;
+    private static final String OPTS = "-showversion";
+    private static final String BUILD_FILE = "custom-build-file.xml";
+    private static final String FAKE_BUILD_FILE = "fake.xml";
+    private static final String SCRIPT_PIPELINE_ANT = "node {\n" +
+            "    withAnt(installation: '" + NATIVE_ANT_NAME + "') {\n" +
+            "        if (isUnix()) {\n" +
```

Comment:
as this is only used in one test and as that test should be excersized on the SSH agent only (never the controller) we know that this will be a unix docker agent (as we do not support windows containers here - and if we did that would be a lot of work!)  
so you can remove the `isUnix` if/esel and just assume it is always unix (after fixing the `node` block to use the approprate agent!